### PR TITLE
Release/1.1.8

### PR DESCRIPTION
**Pull request by AmirQSD**
**Implemented enhancements:**
 
 - Add support to get channel lifecycle status  [\#155](https://github.com/ably/ably-php/issues/155)
 
 **Merged pull requests:**
 
 - Add support for MsgPack [\#161](https://github.com/ably/ably-php/pull/161) ([AmirQSD](https://github.com/AmirQSD))
 - Channel status [\#159](https://github.com/ably/ably-php/pull/159) ([sacOO7](https://github.com/sacOO7))
 - Refactor code [\#152](https://github.com/ably/ably-php/pull/152) ([sacOO7](https://github.com/sacOO7))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+
+[Full Changelog](https://github.com/ably/ably-php/compare/1.1.7...1.1.8)
+
+**Implemented enhancements:**
+
+- Add support to get channel lifecycle status  [\#155](https://github.com/ably/ably-php/issues/155)
+
+**Merged pull requests:**
+
+- Add support for MsgPack [\#161](https://github.com/ably/ably-php/pull/161) ([AmirQSD](https://github.com/AmirQSD))
```
Comment (owenpearson) [inline CHANGELOG.md:13]: I think you should add this one to the implemented enhancements section too

**Review by owenpearson — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (AmirQSD) [reply CHANGELOG.md:13]: OK, keep it in both places then? @owenpearson

Comment (owenpearson) [reply CHANGELOG.md:14]: yeah exactly 👍

**Review by qsdstefan — APPROVED** [review]
Looks good!

**Review by owenpearson — APPROVED** [review]
LGTM, thanks

**Review by sacOO7 — COMMENTED** [review]
LGTM : )

**Pull request merged**